iceoryx2-services/discovery/src/service_discovery/service.rs

```diff
@@ -214,6 +264,12 @@ pub struct Config {
 
     /// The maximum number of listeners to the service permitted.
     pub max_listeners: usize,
+
+    /// Whether to enable the server for handling requests.
+    pub enable_server: bool,
```

Comment:
Not a bad idea. What was your reasoning, though?

Reply:
@RTK-jangid Forgot to ping you here. Could you share your reasoning (just so I am on the same page).